### ISSUE TITLE
NTR: peptide-mediated antimicrobial humoral response

### ISSUE THREAD

**Issue by zebrafishembryo** [geneontology/go-ontology#12961]
Hello, 

I would like to request a new BP term “peptide-mediated antimicrobial humoral response”.  
The possible definition can be “An immune response against microbes mediated by peptides through a body fluid.”  
This term can be a child of GO:0019730 “antimicrobial humoral response”. 
 The main reason for this term request is to be able to capture the experimental data available for the so-called antimicrobial peptides and differentiate it from other types of humoral response (mediated by antibodies or complement proteins), since the MF term “antimicrobial peptide activity” has been obsolete. 

Some useful links: http://www.sigmaaldrich.com/technical-documents/articles/biofiles/antimicrobial-peptides.html
https://www.ncbi.nlm.nih.gov/pubmed/15761415
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3873676/

Many thanks, 
Alice

**Comment by ukemi** [comment]
It almost seemed like the existing term was meant to represent this, but I'm not an immunologist and based just on the definition, it wasn't specific enough. So I have made this new term. 

 [Term]
+id: GO:0061844
+name: antimicrobial humoral immune response mediated by antimicrobial peptide
+namespace: biological_process
+def: "An immune response against microbes mediated by anti-microbial peptides in body fluid." [PMID:15761415, PMID:24287494]
+synonym: "antimicrobial peptide-mediated antimicrobial humoral response" EXACT []
+synonym: "peptide-mediated antimicrobial humoral response" BROAD []
+is_a: GO:0019730 ! antimicrobial humoral response

I also moved the existing antimicrobial peptide production to be a part of it.
